lua/entities/gmod_wire_light.lua

```diff
@@ -245,7 +245,8 @@ function ENT:TriggerInput(iname, value)
 	self:UpdateLight()
 end
 
-function ENT:Setup(directional, radiant, glow, brightness, size, r, g, b, spritesize)
+function ENT:Setup(directional, radiant, glow, brightness, size, r, g, b, spritesize, startOn)
+	self.startOn = startOn == nil or startOn
```

Comment:
Just delete this line and put it in SetOn. `self:SetOn(startOn == nil or startOn)`